packages/venia-concept/src/components/Checkout/addressForm.css

```diff
@@ -54,3 +54,8 @@
 .street0 {
     grid-column-end: span 2;
 }
+
+.validation {
+    grid-column-end: span 2;
+    line-height: normal;
+}
```

Comment:
Can we use any of our [CSS color variables](https://github.com/magento-research/pwa-studio/blob/release/2.0/packages/venia-concept/src/index.css#L6-L19) to make this error stand out a bit more?

![screen shot 2018-12-19 at 10 57 11 am](https://user-images.githubusercontent.com/13182778/50235386-631ab200-037d-11e9-8ea5-42ccda1f3de1.png)

Reply:
this will be a bit out of scope of this issue, changes related to the appearance of address and input are placed in this pr https://github.com/magento-research/pwa-studio/pull/593